### PR TITLE
Update new Popup Manager

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

File: atomic_defi_design/Dex/Components/DexPopupManager.qml [hunk 1]
```diff
@@ -0,0 +1,293 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import Qaterial 1.0 as Qaterial
+import QtQuick.Layouts 1.12
+import App 1.0
+import Dex.Themes 1.0 as Dex
+
+Popup {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexPopupManager.qml:8]: Coding style

File: atomic_defi_design/Dex/Components/DexPopupManager.qml [hunk 2]
```diff
@@ -0,0 +1,293 @@
+import QtQuick 2.15
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexPopupManager.qml:1]: Remove `Dex` prefix from file name

**Review by SylEze — CHANGES_REQUESTED** [review]

**Comment by smk762** [comment]
Figma on the right
image
- button wording
- button state (always enabled in app)
- input field background darkness

Figma on the right
image
- field input icon misaligned, backfround darkness
- fonts
- button state (always enabled in app)

I think "Cancel" is actually a better choice than "close" in the above cases (disagree with figma)

not in figma
image
- "Wallet status" should be bolder
- Change to "{wallet_name} password is incorrect" (remove "entered", it is not needed and reads funny)
- "Yes" should be "Ok"

Figma on right
image
- Font weight and alignment

Figma on right
image
- Font weight and alignment
- No button
- dot count at top

Figma on right
image
- Font weight and alignment
- No button
- dot count at top

Pls let me know if I'm missing anywhere else this is used.

**Comment by ssakone** [comment]
@smk762  not able to find screen 3, and screen 5, 6 is another type of dialogue, let me sync with  syl about that update.

**Comment by smk762** [comment]
Screen 3 is if you input wrong password when attempting to delete wallet

**Comment by ssakone** [comment]
should done but, check title bold level now, i think its not okay

**Comment by ssakone** [comment]
For screen two i think color correction will fix it, @SylEze help!

**Review by tonymorony — CHANGES_REQUESTED** [review]
looks quite acceptable to me, just would be great to use the same color for input fields as other parts of the app, so we'll be able to change it globally across the app for better contrast and/or during other skins creation

Screenshot 2021-11-28 at 23 21 25

e.g.

Screenshot 2021-11-28 at 23 26 23

**Comment by smk762** [comment]
image
Change "Enable" to "Next >"

image
Eyeball on right side of text input is hard to see

Seems when I try and delete wallet, it always says "wrong password", even if I use the password I can successfully log in with.

**Comment by tonymorony** [comment]
app from this https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/1524893773 build not starting for me on OSX:


```
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: QQmlApplicationEngine failed to load component
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/main.qml:411:5: Type App unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/App.qml:85:9: Type Dashboard unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Screens/Dashboard.qml:134:13: Type Wallet unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Wallet/Wallet.qml:54:5: Type Main unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Wallet/Main.qml:347:34: Type SendModal unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Wallet/SendModal.qml:10:1: Type BasicModal unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Components/BasicModal.qml:7:1: Type DefaultModal unavailable
[23:36:09] [warning] [main.prerequisites.hpp:92] [735159]: qrc:/Dex/Components/DefaultModal.qml:12:28: Invalid alias reference. Unable to find id "background"
[23:36:09] [info] [main.prerequisites.hpp:486] [735159]: qml engine successfully loaded
[23:36:09] [error] [main.prerequisites.hpp:106] [735159]: sigabort received, cleaning mm2
No matching processes belonging to you were found
[23:36:09] [error] [main.prerequisites.hpp:112] [735159]: stacktrace:  0# main in /Applications/atomicdex-desktop.app/Contents/MacOS/atomicdex-desktop
 1# _sigtramp in /usr/lib/system/libsystem_platform.dylib
 2# QWindow::winId() const in /Applications/atomicdex-desktop.app/Contents/Frameworks/QtGui.framework/Versions/5/QtGui
 3# main in /Applications/atomicdex-desktop.app/Contents/MacOS/atomicdex-desktop
 4# main in /Applications/atomicdex-desktop.app/Contents/MacOS/atomicdex-desktop
```

**Review by tonymorony — CHANGES_REQUESTED** [review]
can't start, since builds from other branches works I assume some problems with changes

**Comment by smk762** [comment]
> can't start, since builds from other branches works I assume some problems with changes

Also unable to start here, same errors in logs.

**Comment by ssakone** [comment]
> 

should fixed now.

**Review by tonymorony — APPROVED** [review]

**Review by SylEze — APPROVED** [review]

**Pull request merged**